draft-ietf-quic-transport.md

```diff
@@ -1378,28 +1378,39 @@ A server MUST include the original_connection_id transport parameter
 ({{transport-parameter-definitions}}) if it sent a Retry packet to enable
 validation of the Retry, as described in {{packet-retry}}.
 
-
 ### Values of Transport Parameters for 0-RTT {#zerortt-parameters}
 
-A client that attempts to send 0-RTT data MUST remember the transport parameters
-used by the server.  The transport parameters that the server advertises during
-connection establishment apply to all connections that are resumed using the
-keying material established during that handshake.  Remembered transport
-parameters apply to the new connection until the handshake completes and new
-transport parameters from the server can be provided.
-
-A server can remember the transport parameters that it advertised, or store an
-integrity-protected copy of the values in the ticket and recover the information
-when accepting 0-RTT data.  A server uses the transport parameters in
-determining whether to accept 0-RTT data.
-
-A server MAY accept 0-RTT and subsequently provide different values for
-transport parameters for use in the new connection.  If 0-RTT data is accepted
-by the server, the server MUST NOT reduce any limits or alter any values that
-might be violated by the client with its 0-RTT data.  In particular, a server
-that accepts 0-RTT data MUST NOT set values for the following parameters
-({{transport-parameter-definitions}}) that are smaller
-than the remembered value of those parameters.
+Clients remember the value of transport parameters from a server and apply
+them to any 0-RTT packets that are sent in subsequent connections, except
+for transport parameters that are explicitly excluded. Remembered transport
+parameters apply to the new connection until the handshake completes and the
+client starts sending 1-RTT packets. Once the handshake completes, the client
+uses the transport parameters established in thehandshake.
+
+The value of the server's previous original_connection_id, preferred_address,
```

Comment:
Editorial rewrite suggestion:  "A client MUST NOT use remembered values for the following parameters: original_connection_id, preferred_address, stateless_reset_token, ack_delay_exponent.  The client MUST use the server's new values in the handshake instead, and absent new values from the server, the default value."